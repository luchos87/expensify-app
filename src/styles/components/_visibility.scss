.show-for-mobile {
    @media (min-width: $desktop-breackpoint) {
        display: none;
    }
}

.show-for-desktop {
    @media (max-width: ($desktop-breackpoint - .01rem)) {
        display: none;
    }
}