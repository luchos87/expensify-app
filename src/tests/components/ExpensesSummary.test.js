import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('render ExpensesSummary with 1 expenses', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235} />);
    expect(wrapper).toMatchSnapshot(); 
});

test('render ExpensesSummary with multiples expenses', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={231235} />);
    expect(wrapper).toMatchSnapshot(); 
});