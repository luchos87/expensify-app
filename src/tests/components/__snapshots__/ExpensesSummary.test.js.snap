// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ExpensesSummary with 1 expenses 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`render ExpensesSummary with multiples expenses 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $2,312.35
  </h1>
</div>
`;
